poonam/JS/Assignment6_js/que2.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>que2</title>
+        <script>
+            var i=0;
```

Comment:
there should be space before and after any operator like " =" operator or +, - and all type operator do this change in all files and space should be there after comma, not before